overview.md

```diff
@@ -22,19 +22,19 @@ Open Source Discovery: Rapid scanning and identification of open source librarie
 
 Detect for Azure DevOps is architected to integrate seamlessly with build and release pipelines for Black Duck
 
-![extension](images/new-extension.png)
+![extension](images/bd-extension.png)
 
 ## Documentation ##
 
-Instructions and examples are available on our [Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Synopsys+Detect+for+Azure+DevOps)
+Instructions and examples are available on our [Public Confluence](https://blackduck.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Black+Duck+Detect+for+Azure+DevOps)
 
-For information on the full capabilities of Detect visit [Detect Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Synopsys+Detect)
+For information on the full capabilities of Detect visit [Detect Public Confluence](https://blackduck.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Black+Duck+Detect)
 
 ## Pre-Requisites ##
 
 Before calling Detect in Azure DevOps, an active instance of Black Duck is required.
 
-If you do not have Black Duck, refer to [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/black-duck-software.blackduck_hub_431) for more information.
+If you do not have Black Duck, refer to [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/blackduck-software.blackduck_hub_431) for more information.
```

Comment:
Just Confirmed, the new link that was updated is not correct. This is the link which will work: https://azuremarketplace.microsoft.com/en-us/marketplace/apps/black-duck-software.blackduck_hub_431

Reply:
That link points to some (outdated?) "Black Duck by Synopsys": 
![Untitled](https://github.com/user-attachments/assets/de19a308-1eb4-4508-84a5-3ac767084680)

Reply:
I see, but the previous one is a broken link so for now its better to point to the old version. I believe the team maintaining it would release a new rebranded version but have to check with them.